CHANGELOG.rst

```diff
@@ -16,6 +16,10 @@ Please See the `releases tab <https://github.com/openedx/xblock-lti-consumer/rel
 Unreleased
 ~~~~~~~~~~
 
+9.5.0 - 2023-05-08
+------------------
+* Return HTML message telling user that exam is ready to start
```

Comment:
NIT: I'd recommend making this a bit more specific it's clear this is for the start assessment response to a proctored exam launch.